docs/user/subprojects.rst

```diff
@@ -1,43 +1,67 @@
+* Subproject: https://example-project.readthedocs.io/projects/plugin/en/latest/
 
-In the admin dashboard for your project, select "Subprojects" from the menu.
-From this page you can add a subproject by typing in the project slug.
+.. seealso::
 
-Subproject aliases
-~~~~~~~~~~~~~~~~~~
+   :doc:`/guides/subprojects`
+     Learn how to create and manage subprojects
+
+   :doc:`/guides/intersphinx`
+     Learn how to use references between different Sphinx projects, for instance between subprojects
 
-You can use an alias for the subproject when it is created. This allows you to override the URL that is used to access it, giving more configurability to how you want to structure your projects.
 
 Sharing a custom domain
 -----------------------
 
-Projects and subprojects can also be used to share a custom domain with a number
-of projects. To configure this, one project should be established as the parent
-project. This project will be configured with a custom domain. Projects can then
-be added as subprojects to this parent project.
+Projects and subprojects can be used to share a custom domain.
+To configure this, one project should be established as the main project.
+This project will be configured with a custom domain.
+Other projects are then added as subprojects to the main project.
+
+If the example project ``example-project`` was set up with a custom domain,
+``docs.example.com``, the URLs for projects ``example-project`` and ``example-project-plugin`` with alias ``plugin`` would
+respectively be at:
+
+* ``example-project``: https://docs.example.com/en/latest/
+* ``example-project-plugin``: https://docs.example.com/projects/plugin/en/latest/
+
+Using aliases
+-------------
+
+You can choose an alias for the subproject when it is created.
+This allows you to override the URL that is used to access it,
+giving more control over how you want to structure your projects.
+
+You can set your subproject's project name and :term:`slug` however you want,
+but we suggest prefixing it with the name of the main project.
 
-If the example project ``Foo`` was set up with a custom domain,
-``docs.example.com``, the URLs for projects ``Foo`` and ``Bar`` would
-respectively be at: https://docs.example.com/en/latest/ and
-https://docs.example.com/projects/bar/en/latest/
+Typically, a subproject is created with a ``$mainproject-`` prefix,
+for instance if the main project is called ``example-project-plugin`` and the subproject is called ``plugin``,
+then the subproject's Read the Docs project name will be ``example-project-plugin``.
+When adding the subproject,
+the alias is set to ``plugin`` and the project's URL becomes
+``example-project.readthedocs.io/projects/plugin``.
+
+When you add a subproject,
+the subproject will not be directly available anymore from its own domain.
+For instance, ``example-project-plugin.readthedocs.io/`` will redirect to ``example-project.readthedocs.io/projects/plugin``.
```

Comment:
It will still be available from it owns domain, but it will redirect.

```suggestion
Read the Docs will redirect the old subproject URL to the new one using the main project domain and the alias in the path.
For instance, ``example-project-plugin.readthedocs.io/`` will redirect to ``example-project.readthedocs.io/projects/plugin``.
```

Reply:
I get it now, saying "not be directly available" contradicts the redirect, since a redirect strictly speaking makes it available :+1: